src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationHandler.cs

```diff
@@ -217,6 +218,47 @@ public OpenIdConnectAuthenticationHandler(HttpClient backchannel)
 
             message.State = Options.StateDataFormat.Protect(properties);
 
+            if (Options.UsePostAuthorizationRequests)
+            {
+                using (var buffer = new MemoryStream())
+                using (var writer = new StreamWriter(buffer))
+                {
+                    writer.WriteLine("<!doctype html>");
+                    writer.WriteLine("<html>");
+                    writer.WriteLine("<body>");
+
+                    writer.WriteLine("<form name='form' method='post' action='" + Options.HtmlEncoder.HtmlEncode(message.IssuerAddress) + "'>");
```

Comment:
The last time I tried `IHtmlEncoder` (well, its default implementation), using it with single quotes was safe.
@blowdart 'zat still true?